src/reducers/books.js

```diff
@@ -0,0 +1,26 @@
+const booksReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'CREATE_BOOK':
+      return [
+        ...state,
+        {
+          id: action.id,
+          title: action.title,
+          category: action.category,
+        },
+      ];
+    case 'REMOVE_BOOK':
```

Comment:
- [ ] How is this different from `CREATE_BOOK` case? This should remove the book with the id you want to remove. You can use `filter` method to filter the books.